server/supernova/audio_backend/jack_backend.hpp

```diff
@@ -23,6 +23,11 @@
 #include <string>
 #include <vector>
 
+
+#ifdef _WIN32
+#define _PTHREAD_H
```

Comment:
can you explain why this is here? I can't find any information about this macro

Reply:
In jack include files (systemdeps.h) this allows
<code>typedef pthread_t jack_native_thread_t;</code>

Reply:
https://github.com/jackaudio/jack2/blob/master/common/jack/systemdeps.h#L91

Reply:
this should go in a comment

Reply:
I have not been able to find such documentation :-(

Reply:
thanks! I appreciate that. Either way, just putting a comment above this to explain it's defined to force jack to use pthread as the native handle would be lovely

Reply:
In my last commit "use GetCurrentThread" I am proceeding as advised in Jack2 repo. Althought this advise is not completely mandatory as can be seen in the second post.